spec/system/admin_about_config_area_spec.rb

```diff
@@ -105,7 +105,8 @@
       expect(config_area.general_settings_section.banner_image_uploader).to have_uploaded_image
 
       config_area.general_settings_section.banner_image_uploader.toggle_lightbox_preview
-      expect(config_area.general_settings_section.banner_image_uploader).to have_lighbox_preview
+      expect(config_area.general_settings_section.banner_image_uploader).to have_lightbox_preview
+      config_area.general_settings_section.banner_image_uploader.close_lightbox_preview
```

Comment:
We have to close the lightbox preview as it is blocking the submit button. This problem was hidden previously because we were using javascript to submit the form.